<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1911</width>
      <height>1041</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_camera">
     <attribute name="title">
      <string>Camera</string>
     </attribute>
     <widget class="MyLabel" name="label_camera_image">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_grab">
      <property name="geometry">
       <rect>
        <x>1200</x>
        <y>100</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Grab</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_data_path">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>100</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_save_path">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>110</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Save Path :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_save_image_lists">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>210</y>
        <width>631</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_data_counts">
      <property name="geometry">
       <rect>
        <x>1370</x>
        <y>210</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>760</y>
        <width>340</width>
        <height>112</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButton_blur">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="text">
          <string>blur</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_blur</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="radioButton_gaussianBlur">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>GaussianBlur</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_blur</string>
         </attribute>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="radioButton_bilateraFilter">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>bilateralFilter</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_blur</string>
         </attribute>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="radioButton_medianBlur">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="text">
          <string>medianBlur</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_blur</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="radioButton_blur_none">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>none</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_blur</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBox_blur_num">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_train">
     <attribute name="title">
      <string>Train</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_train_flag">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Train</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_train_log">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>611</width>
        <height>661</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_train_result">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>170</y>
        <width>581</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_test">
     <attribute name="title">
      <string>Test</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_test">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>0</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_model_path">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Model Path:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_model_path">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_test_camera_image">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="text">
       <string>camera image</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_defect_result">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>100</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="text">
       <string>defect image</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_defect_threshold">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Defect Threshold:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_defect_threshold">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>10</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_block_size">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>30</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Block Size:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_block_size">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>10</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QChartView" name="line_chart_view">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>581</y>
        <width>871</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_all_info">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>580</y>
        <width>551</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_blur"/>
 </buttongroups>
</ui>
